omod/src/main/webapp/resources/js/view/bill.js

```diff
@@ -294,7 +294,7 @@ define(
 				return this.cashPointForm;
 			},
 
-			updateTotals: function() {
+			updateTotals: function () {
```

Comment:
No space between the function and the (). You should update your javascript style to reflect this.